docs/framework/release-notes/2025/03-25-march-cumulative-update-preview.md

```diff
@@ -0,0 +1,44 @@
+---
+title: March 2025 cumulative update preview
+description: Learn about the improvements in the .NET Framework March 2025 cumulative update preview.
+ms.date: 03/27/2025
+---
+# March 2025 cumulative update preview
+
+_Released March 25, 2025_
+_Updated March 27, 2025_
+
+## Summary of what's new in this release
+
+- [Security improvements](#security-improvements)
+- [Quality and reliability improvements](#quality-and-reliability-improvements)
+
+### Security improvements
+
+There are no new security improvements in this release. This update is cumulative and contains all previously released security improvements.
+
+### Quality and reliability improvements
+
+#### CLR
+
+Addresses an issue where System.IO APIs are used to operate on files and directories on UNC shares. (*Applies to: .NET Framework 4.8, 4.8.1*)
+
+## Known issues in this release
+
+This release contains no known issues.
+
+## Summary tables
+
+The following table outlines the updates in this release.
+
+| Product version | Cumulative update preview |
+| --- | --- |
+| **Windows 11, version 24H2** | |
+| .NET Framework 3.5, 4.8.1 | [5054979](https://support.microsoft.com/kb/5054979) |
+| **Windows 11, version 22H2 and Windows 11, version 23H2** | |
+| .NET Framework 3.5, 4.8.1 | [5054980](https://support.microsoft.com/kb/5054980) |
+| **Windows 10, version 22H2** | **[5055063](https://support.microsoft.com/kb/5055063)** |
+| .NET Framework 3.5, 4.8 | [5054977](https://support.microsoft.com/kb/5054977) |
+| .NET Framework 3.5, 4.8.1 | [5054978](https://support.microsoft.com/kb/5054978) |
+
+The operating system row lists a KB that's used for update offering purposes. When the operating system KB is offered, the applicability logic determines the specific .NET Framework update(s) installed. Updates for individual .NET Framework versions are installed based on the version of .NET Framework that is already present on the device. Because of this, the operating system KB is not expected to be listed as installed updates on the device. The expected updates installed are the .NET Framework-specific version updates listed in the previous table.
```

Comment:
```suggestion
The operating system row lists a KB that's used for update offering purposes. When the operating system KB is offered, the applicability logic determines the specific .NET Framework update(s) installed. Updates for individual .NET Framework versions are installed based on the version of .NET Framework that is already present on the device. Because of this, the operating system KB is not expected to be listed as installed updates on the device. The expected updates installed are the .NET Framework-specific version updates listed in the previous table.

```